### PR TITLE
More configurability

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

Make more features configurable and add support for link type columns.

Related to : https://github.com/upfluence/backlog/issues/696

### 🧑‍💻 Developer Heads Up

⚡ Since we are using [Ember Octane](https://blog.emberjs.com/octane-is-here/) now:
* Feel free to migrate existing components to Glimmer Components.
* Write new ones exclusively in it.

Useful Resource : [Ember Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

**Review by Hegazy360 — APPROVED** [review]

File: addon/components/hyper-table/cell.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default Component.extend({
 
     if (this.manager.hooks.onRowClicked && !editStatus && !this.selection && !isAutosave)
       this.manager.hooks.onRowClicked(
-        this.item.influencer_id, (this.column || {}).key
+        this.item, (this.column || {}).key
```
Comment (phndiaye) [inline addon/components/hyper-table/cell.js:26]: Breaking Change

File: app/templates/components/hyper-table.hbs [hunk 2]
```diff
@@ -34,15 +34,17 @@
...
-      </span>
-    </button>
+    {{#if manager.options.features.tableViews}}
+      <button
+        class="margin-right-xx-sm margin-bottom-xxx-sm upf-btn upf-btn--default upf-btn--small views-manager-toggle icon-responsive"
+        data-control-name="table_toggle_views"
+        {{action "openAvailableViews" bubbles=false}}>
+        <i class="fa fa-eye"></i>
+        <span>
+          &nbsp; Table Views
```
Comment (JulienVannier66) [inline app/templates/components/hyper-table.hbs:44]: Can you use missing translation in this file?

File: app/templates/components/hyper-table/cell-renderers/image.hbs [hunk 3]
```diff
@@ -5,7 +5,7 @@
 {{#if (gt column.labels.length 0)}}
   <div>
     {{#each column.labels as |label|}}
-      <span data-toggle="tooltip" data-title={{get item label}}>
+      <span class="margin-left-xx-sm" data-toggle="tooltip" data-title={{get item label}}>
```
Comment (JulienVannier66) [inline app/templates/components/hyper-table/cell-renderers/image.hbs:8]: Can you use tooltip modifier?

File: app/styles/cells.less [hunk 4]
```diff
@@ -233,6 +234,30 @@
...
+    white-space: nowrap;
+  }
+
+  .link__copy {
+    position: relative;
+    display: flex;
+    align-items: center;
+    background-color: @upf-gray-light;
+    height: 30px;
+    padding: 0.5em;
```
Comment (JulienVannier66) [inline app/styles/cells.less:254]:
```suggestion
    padding: @spacing-xxx-sm;
```

File: app/templates/components/hyper-table/column.hbs [hunk 5]
```diff
@@ -34,14 +33,16 @@
 
   <div class="icon-commands" data-control-name={{concat "table_column_filter_sort_toggle_" column.key}}>
     {{#if _supportsOrderingOrFiltering}}
-      <i data-toggle="tooltip" data-title='Filters' data-placement="bottom" class="fa fa-filter" {{action "toggleFiltersPanel"}}></i>
+      <i class="fa fa-filter" {{enable-tooltip title="Filters"}} {{action "toggleFiltersPanel" bubbles=false}}></i>
```
Comment (JulienVannier66) [inline app/templates/components/hyper-table/column.hbs:36]: can you use translation for tooltip title?

**Review by Hegazy360 — APPROVED** [review]

**Review by JulienVannier66 — APPROVED** [review]
LGTM

**Pull request merged**